paimon-web-gateway/src/main/java/org/apache/paimon/web/gateway/provider/ExecutorFactoryProvider.java

```diff
@@ -34,7 +34,7 @@ public ExecutorFactoryProvider(ExecutionConfig executionConfig) {
 
     public ExecutorFactory getExecutorFactory(EngineType type) {
         switch (type) {
-            case FLINK:
+            case FLINK_SQL_GATEWAY:
```

Comment:
Is this change related with cluster status check?

Reply:
To more clearly identify different Flink engines, in the cluster management page, Flink currently has session and sql gateway cluster types. This issue has been unified in the WeChat group a few days ago.